sql/moz-fx-data-shared-prod/telemetry_derived/error_aggregates_v1/metadata.yaml

```diff
@@ -4,7 +4,7 @@ description: |-
   Counts of various error measures aggregated across each unique
   set of dimensions.
 owners:
-  - wlachance@mozilla.com
+  - wkahngreene@mozilla.com
```

Comment:
Needed to update this in the query metadata in order to get the correct generated output.